pyscriptjs/src/main.ts

```diff
@@ -160,6 +160,15 @@ export class PyScriptApp {
             );
         }
         this.config = loadConfigFromElement(el);
+        if (this.config.execution_thread === 'worker' && crossOriginIsolated === false) {
+            throw new UserError(
+                ErrorCode.BAD_CONFIG,
+                `execution_thread is set to "worker" but the following CORS headers are missing: ${JSON.stringify({
+                    'Cross-Origin-Embedder-Policy': 'require-corp',
+                    'Cross-Origin-Opener-Policy': 'same-origin',
+                })}`,
```

Comment:
How do we know this? There might be a different reason that `crossOriginIsolated` is `false`, like the site being served over http (not https) or a browser bug. In general error messages should only make assertions of facts that we **know** are true. The reason that we are throwing the error is because crossOriginIsolated is false. So this is what we should report. We can then follow that with a suggestion that we think most likely because we are missing one or both of these two headers.

The following is more in the direction I'd think:
```
When execution_thread is "worker", the site must be cross origin isolated 
but crossOriginIsolated is false. To be cross origin isolated, it must be served with the following headers:

The problem may be that one or both of these are missing.
[in followup PR add link to discussion in DOCs]
```

Reply:
Also, in Chrome we have the option to set 
```
Cross-Origin-Embedder-Policy: credentialless
```
which is quite exciting because it allows you to load static resources from places that don't have the cross origin resource policy set. But it disabled authenticated requests so that's a tradeoff.